src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp

```diff
@@ -39,6 +41,18 @@
 /* ****************************************************************************
 *
 * getAllEntitiesWithTypeAndId - 
+*
+* GET /v1/contextEntities/type/{entity::type}/id/{entity::id}
+* GET /ngsi10/contextEntities/type/{entity::type}/id/{entity::id}
+*
+* Payload In:  None
+* Payload Out: ContextElementResponse
+* 
+* URI parameters:
+*   - attributesFormat=object
```

Comment:
Yeah, it's only one output from this request (ContextElementResponse - not a vector).
There is nothing to count.